### DISCUSSION TITLE
IsEmpty No answer, try again

### DISCUSSION THREAD

**Discussion by PeterPanino** [AliDehbansiahkarbon/ChatGPTWizard#45] [Q&A]
When clicking the ASK button, I always get this answer: IsEmpty No answer, try again
Of course, the correct API key has been entered.
There is no verification feedback on the API key.

**Comment by AliDehbansiahkarbon** [comment]
This means the response content of the API is empty, there is no error with the Post method just there is an empty response.
Something is wrong with the API it seems, try with a different model in the setting form.

![image](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/assets/5601608/9d778928-e4b1-4d31-9f0d-5019c7a9cc98)
